app/menu.js

```diff
@@ -1,11 +1,15 @@
+'use strict';
```

Comment:
To use ES6 features on node properly it is required
e.g. my use of `for (const shellName of Object.keys(shells)) {` will break without it. Other than node 7 the above behaviour change with or without `use strict`. Without it `shellName` is set on the first iteration and remains constant for all other iterations. With it shellName is a constant to the scope of the loops inner block, however changes properly with each iteration.